pkgs/tools/backup/zfsbackup/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoPackage rec {
   pname = "zfsbackup";
-  version = "unstable-2019-03-05";
-  rev = "78fea6e99f0a5a4c8513d3a3d1d45fb6750cfddf";
+  version = "unstable-2020-11-02";
```

```suggestion
  version = "unstable-2020-09-30";
```

May as well match a date of last commit.

---

Sure! will push a new update with the suggested change.